ulog/README.md

```diff
@@ -1,16 +1,15 @@
 # Logging package
 
 `package ulog` provides an API wrapper around the logging library
```

Comment:
We still provide ulog.Logger(ctx) which is a major use case explained in context logging.